tests/scripts/fluentd_helm_e2e.sh

```diff
@@ -0,0 +1,78 @@
+PROJECT_ROOT=$PWD
+E2E_DIR=$(realpath $(dirname $0)/..)
+LOGGING_NAMESPACE=kubesphere-logging-system
+IMAGE_TAG=latest
+
```

Comment:
latest should be modified to other values, see https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster. Former codes ignoring this case, please help to fix it.
> can be `date +"%Y-%m-%d"`

Reply:
The other script need to be fixed also.